python_frank_energie/frank_energie.py

```
@@ -18,25 +20,57 @@
 from .authentication import Authentication
-from .exceptions import (AuthException, AuthRequiredException,
-                         ConnectionException, FrankEnergieError,
-                         FrankEnergieException, LoginError, NetworkError,
-                         RequestException, SmartTradingNotEnabledException, SmartChargingNotEnabledException)
-from .models import (Authentication, EnergyConsumption, EnodeChargers, Invoice, Invoices,
-                     MarketPrices, Me, MonthInsights, MonthSummary,
-                     PeriodUsageAndCosts, SmartBatteries, SmartBattery, SmartBatteryDetails, SmartBatterySummary, SmartBatterySessions, User, UserSites)
+from .exceptions import (
+    AuthException,
+    AuthRequiredException,
+    ConnectionException,
```

Ruff (0.12.2)

`.exceptions.ConnectionException` imported but unused

Remove unused import

(F401)